packages/starlight/user-components/LinkCard.astro

```diff
@@ -11,25 +11,36 @@ const { title, description, ...attributes } = Astro.props;
 ---
 
 <div>
```

Comment:
Not an a11y expert but could we use an `<article>` instead of a `<div>` for a more semantic a11y tree.

Reply:
i like that. articles need should generally have labels though.
```html
<article aria-labelledby='title'>
  <a id='title'>...</a>
```

is there a utility i can use for generating a unique id?